theme/common.less

```diff
@@ -361,11 +355,75 @@ div.simframe > iframe {
     padding: 0.3em;
 }
 
+/*******************************
+      Windows store banner
+*******************************/
 /* Experimental banner */
-#experimentalBanner {
+#experimentalBanner, #windowsStoreBanner {
     z-index: 1000;
 }
 
+.showWindowsStoreBanner #filelist,
+.showWindowsStoreBanner #maineditor,
+.showWindowsStoreBanner #sidedocs {
+    top: calc(@mainMenuHeight + @bannerHeight);
+}
+
+.showWindowsStoreBanner #mainmenu {
+    top: @bannerHeight;
+    margin: 0;
+}
+
+#windowsStoreBanner {
+    padding: 0;
+    background-color: @bannerBackgroundColor;
+    color: @bannerContrastColor;
+    outline: 2px solid @bannerContrastColor;
+    font-family: "Segoe UI", "Helvetica Neue", sans-serif;
+    height: @bannerHeight;
+    margin-top: 0;
+    img {
+        height: calc(@bannerHeight - 0.5rem);
+        margin: 0 0.5rem 0 0.5rem;
+    }
+    a.link {
+        text-decoration: none;
+        color: @bannerContrastColor;
+        height: 100%;
+        float: left;
+
+        &:hover, &:focus {
+            text-decoration: none;
+            color: @linkColor;
+        }
+        span {
+            height: 100%;
+            display: flex;
+            align-items: center;
+            float: left;
+        }
+    }
+    .bold {
+        font-weight: bold;
+    }
+    .close {
+        float: right;
+        line-height: @bannerHeight;
+        cursor: pointer;
+        &:hover, &:focus {
+            outline: none;
+            color: darken(@bannerContrastColor, 25%);
+        }
+    }
+}
+
+#windowsStoreBanner
```

Comment:
No need to move this. Keep this in the Layout section. (at the top)

Reply:
I'd almost go as far as just removing it from this list and adding a top: 0 in it's own #windowStoreBanner style definition. This bit of CSS is for hiding the menu bar.

Reply:
Don't understand the request - you want me to move this one rule back to the Layout section?  Or all windowsStoreBanner rules?